docs/source/user_guide/automl.ipynb

```diff
@@ -325,7 +336,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
```

Comment:
Will revert this and for other files.

Reply:
Lol yeah... I think its fine that these vary a bit. Not sure what we should do about it yet, its annoying for us to always have to change/un-change them! Will file.

Reply:
https://github.com/alteryx/evalml/issues/1419

Reply:
Agreed; thanks for filing!